hedera-node/hedera-file-service-impl/src/main/resources/genesis/simpleFeesSchedules.json

```diff
@@ -72,28 +72,28 @@
       "schedule": [
         {
           "name": "ConsensusCreateTopic",
-          "baseFee": 99700000,
+          "baseFee": 99000000,
           "extras": [
             { "name": "KEYS", "includedCount": 0 }
           ]
         },
         {
           "name": "ConsensusUpdateTopic",
-          "baseFee": 3240000,
+          "baseFee": 2100000,
```

Comment:
Oh may be I was wrong. This should be 1200000 ? 

((0.00022 * 10**10) - 1000000) Am I right @joshmarinacci